homework/07LessonSeven/script.js

```diff
@@ -69,20 +69,22 @@ $(document).on('ready', function() {
   // Problem 9:
   // select the element with id="insert-text"
   // then set the text of this element to "Text Has Been Inserted"
+  $('#insert-text').text('Text Has Been Inserted');
```

Comment:
Good work on this homework and app6!! Hows OOP? Always read the tests to see why something that definitely should be passing isnt!